src/components/ui/Image/ImageWithArtDirection.tsx

```diff
@@ -1,42 +1,27 @@
 import { useGetThumborImageData } from '@vtex/gatsby-plugin-thumbor'
 import { GatsbyImage, withArtDirection } from 'gatsby-plugin-image'
 import React, { useMemo } from 'react'
-import imagesConf from 'src/images/config'
+import type { ThumborImageOptions } from '@vtex/gatsby-plugin-thumbor'
 import type { GatsbyImageProps } from 'gatsby-plugin-image'
 
 interface Props extends Omit<GatsbyImageProps, 'image'> {
-  desktop: {
-    src: string
-    variant: string
-  }
-  mobile: {
-    src: string
-    variant: string
-  }
+  desktop: ThumborImageOptions
+  mobile: ThumborImageOptions
 }
 
 function ImageWithArtDirection({ mobile, desktop, ...imgProps }: Props) {
```

Comment:
Out of curiosity, what is the purpose of this `ImageWithArtDirection` component?

Reply:
take a look in here: https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#withartdirection